### ISSUE TITLE
Polygon tap targets are inconsistent

### ISSUE THREAD

**Issue by floatplane** [NWACus/avy#48]
When trying to tap Snoqualmie Pass, I'm getting about a 50% hit rate - the other 50% seem to be divided between West Slopes South and West Slopes Central.

I found an _excellent_ bug report here that seems to describe a similar problem: https://github.com/react-native-maps/react-native-maps/issues/3280. The bug was never addressed, and ultimately closed.

I think we may need to do some kind of `point-in-polygon` check as described in that bug report.

**Comment by stevekuznetsov** [comment]
Yep, sorry, should have written this down - found it way back in the day. Look at the usage of `d3.geoContains()` for finding the zone that pertains to an observation - we will want that logic for taps as well, but likely that will force us to figure out why that method seems buggy - I worry the polygon definitions are somehow leaky.

**Comment by yuliadub** [comment]
I feel like from playing around with this, the tapping seems to be working much better then I remember and I am not sure there is anything to fix here, so closing until someone complains. (my guess maybe the library we are using got better at this or we changed something in the polygons 🤷 )
